Sources/PrivoSDK/api/Rest.swift

```diff
@@ -148,22 +172,36 @@ class Rest {
     }
     
     func trackPossibleAFError(_ error: AFError?, _ response: String?, _ code: Int?) {
+        let encoder = JSONEncoder()
+        var analyticErrorEvent: AnalyticEventErrorData? = nil
```

Comment:
can we avoid using mutating variable here ?

Reply:
yea, will fix it